composer.json

```diff
@@ -13,12 +13,17 @@
     "autoload": {
         "psr-4": {"Utopia\\Config\\": "src/Config"}
     },
+    "scripts": {
+        "lint": "./vendor/bin/pint --test",
+        "format": "./vendor/bin/pint"
+    },
     "require": {
         "php": ">=7.4"
     },
     "require-dev": {
         "phpunit/phpunit": "^9.3",
-        "vimeo/psalm": "4.0.1"
+        "vimeo/psalm": "4.0.1",
+        "laravel/pint": "dev-main"
```

Comment:
Let's use the latest version instead of using the master branch 
https://github.com/laravel/pint/releases/tag/v1.2.0